### PR TITLE
Fix errors in SARIF format

### PR DESCRIPTION
**Pull request by damiencarol**
Fix:

- [X] fix schema url
- [X] add "version"
- [X] add an URL to the vulnerability in Aqua website (not all data are in the file)

### CODE REVIEW

File: contrib/sarif.tpl [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
+  "$schema": "https://raw.githubusercontent.com/oasis-tcs/sarif-spec/master/Schemata/sarif-schema-2.1.0.json",
   "version": "2.1.0",
   "runs": [
     {
       "tool": {
         "driver": {
           "name": "Trivy",
+          "informationUri": "https://github.com/aquasecurity/trivy",
           "fullName": "Trivy Vulnerability Scanner",
+          "version": "1.5.0",
```
Comment (simar7) [inline contrib/sarif.tpl:11]: I assume this is the version of the scanning tool, in this case Trivy. So shouldn't this be `v0.15.0`?

**Review by simar7 — CHANGES_REQUESTED** [review]
thanks for the PR - left a comment but overall looks good.

File: contrib/sarif.tpl [hunk 2]
```diff
@@ -1,12 +1,14 @@
 {
-  "$schema": "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.4.json",
+  "$schema": "https://raw.githubusercontent.com/oasis-tcs/sarif-spec/master/Schemata/sarif-schema-2.1.0.json",
```
Comment (simar7) [inline contrib/sarif.tpl:2]: What has changed between these two versions? Just for my own curiosity how did you know to pick the new one?

Comment (damiencarol) [reply contrib/sarif.tpl:2]: The previous one was a previous working draft. 
The SARIF format is now pretty stable. The specification **fix** it to this value.
Reference: [https://docs.oasis-open.org/sarif/sarif/v2.1.0/cs01/sarif-v2.1.0-cs01.html#_Toc16012437](https://docs.oasis-open.org/sarif/sarif/v2.1.0/cs01/sarif-v2.1.0-cs01.html#_Toc16012437)

File: contrib/sarif.tpl (hunk 1 above)
Comment (damiencarol) [reply contrib/sarif.tpl:11]: To be honest I wanted to rely on a template variable at first but there is no metadata available. Only data from vulnerabilities.
So...

**Comment by damiencarol** [comment]
@simar7 fixed your comment

**Review by simar7 — APPROVED** [review]
thanks lgtm!

**Pull request merged**